tests/jballerina-unit-test/src/test/resources/test-src/statements/ifelse/type-guard.bal

```diff
@@ -923,7 +923,7 @@ function testTypeGuardForCustomErrorPositive() returns [boolean, boolean] {
 
         Details m1 = e5.detail();
         Details m2 = e6.detail();
-        isSpecificError = e5.message() == ERR_REASON && e6.message() == ERR_REASON_TWO && m1 == d && m2 == d;
+        isSpecificError = e5.message() == ERR_REASON && e6.message() == ERR_REASON_TWO;
```

Comment:
Should we check the "message" field in the detail instead of what we are removing?